### PR TITLE
Add Azure portal dashboard

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## Description

This PR adds an Azure portal dashboard for the charges domain. It is a first step towards ensuring working (running) software. It will be expanded and improved over time.

The template JSON was created by first creating the dashboard manually in the Azure Portal, then downloading it (as JSON) and then extracting the lenses-part of the file. Finally, substitution placeholders were added.

The dashboard should look like on the image. The availability test probably needs to either be fixed or removed in the future.

image

## References

[Service-bus-metrics-azure-monitor](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-metrics-azure-monitor#request-metrics)

* #174

### CODE REVIEW

**Comment by prtandrup** [comment]
I'm puzzled by the two count options (active messages and dead letters) displaying decimals, any idea what that is?

File: build/infrastructure/dash-charges-template.json [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+                          },
+                          "name": "IncomingMessages",
+                          "aggregationType": 1,
+                          "namespace": "microsoft.servicebus/namespaces",
+                          "metricVisualization": {
+                            "displayName": "Incoming Messages"
+                          }
+                        }
+                      ],
+                      "title": "Incoming Messages (sum))",
```
Comment (prtandrup) [inline build/infrastructure/dash-charges-template.json:59]: If it was the first time I saw the dashboard, this (or something similar) would be a help to me.
```suggestion
                      "title": "Incoming Messages (sum per type))",
```

Comment (BjarkeMeier) [reply build/infrastructure/dash-charges-template.json:59]: If you want that I guess you would also want (1) "sum per event type" or "sum per topic" and (2) the same addition in all 3 titles. Or...? Personally, I prefer it the way it is but I can change it.

**Comment by BjarkeMeier** [comment]
> I'm puzzled by the count option for active messages displaying decimals, any idea what that is?
> And why is there an "m" after "charge-command-received" in active messages (count)

To be honest: I don't know. I didn't find the answer quickly and think I'd prefer to simply accept it for now considering the (low) priority of the story.

File: build/infrastructure/dash-charges-template.json (hunk 1 above)
Comment (prtandrup) [reply build/infrastructure/dash-charges-template.json:59]: I can live with it as is. Just a thought :)

**Comment by prtandrup** [comment]
> > I'm puzzled by the count option for active messages displaying decimals, any idea what that is?
> > And why is there an "m" after "charge-command-received" in active messages (count)
> 
> To be honest: I don't know. I didn't find the answer quickly and think I'd prefer to simply accept it for now considering the (low) priority of the story.

Agree, but at some point I would like to verify the figures displayed :)

**Review by bemwen — APPROVED** [review]
LGTM

**Review by prtandrup — APPROVED** [review]

File: build/infrastructure/dash-charges-template.json (hunk 1 above)
Comment (BjarkeMeier) [reply build/infrastructure/dash-charges-template.json:59]: Thx for input. I'll however leave it as-is.

**Pull request merged**